test/getCloseTag.test.ts

```diff
@@ -0,0 +1,61 @@
+import { readFileSync } from "fs";
```

Comment:
Need to enable test code running,  similar to  https://github.com/formulahendry/vscode-auto-rename-tag/blob/3141e782cadbb121b8c0fbf13683a88374e6a1b1/package.json#L54

Reply:
Also need to follow the test structure: https://github.com/formulahendry/vscode-auto-close-tag/blob/master/test/extension.test.ts

Reply:
will try to add tests, (code-runner would need tests too, if I figure a good way, I'd update that project too)